services/py-api/src/server/routes/utility_routes.py

```diff
@@ -2,7 +2,7 @@
 
 from src.database.db_manager import DB_MANAGER
 from src.server.handlers.utility_hanlders import UtilityHandlers
-from src.server.schemas.response_schemas.schemas import PongResponse, ErrResponse
+from src.server.schemas.response_schemas.schemas import PongResponse, ErrResponse, NumberOfRandomParticipantsResponse
```

Comment:
Remove the last import NumberOfRandomParticipantsResponse

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4